perses/annihilation/new_relative.py

```diff
@@ -0,0 +1,1351 @@
+        custom_core_force.addPerTorsionParameter('phase2') # molecule2 phase
+        custom_core_force.addPerTorsionParameter('K2') # molecule2 spring constant
+
+        #add the force to the system
+        self._hybrid_system.addForce(custom_core_force)
+        self._hybrid_system_forces['core_torsion_force'] = custom_core_force
+
+        #create and add the torsion term for unique/environment atoms
+        standard_torsion_force = openmm.PeriodicTorsionForce()
+        self._hybrid_system.addForce(standard_torsion_force)
+        self._hybrid_system_forces['standard_torsion_force'] = standard_torsion_force
+
+    def _add_nonbonded_force_terms(self):
+        """
+        Add the nonbonded force terms to the hybrid system. Note that as with the other forces,
+        this method does not add any interactions. It only sets up the forces.
+
+        Parameters
+        ----------
+        nonbonded_method : int
+            One of the openmm.NonbondedForce nonbonded methods.
+        """
+        # Create a CustomNonbondedForce to handle alchemically interpolated nonbonded parameters.
+        # Select functional form based on nonbonded method.
+        if self._nonbonded_method in [openmm.NonbondedForce.NoCutoff]:
+            sterics_energy_expression, electrostatics_energy_expression = self._nonbonded_custom_nocutoff()
+        elif self._nonbonded_method in [openmm.NonbondedForce.CutoffPeriodic, openmm.NonbondedForce.CutoffNonPeriodic]:
+            epsilon_solvent = self._old_system_forces['NonbondedForce'].getReactionFieldDielectric()
+            r_cutoff = self._old_system_forces['NonbondedForce'].getCutoffDistance()
+            sterics_energy_expression, electrostatics_energy_expression = self._nonbonded_custom_cutoff(epsilon_solvent, r_cutoff)
+        elif self._nonbonded_method in [openmm.NonbondedForce.PME, openmm.NonbondedForce.Ewald]:
+            [alpha_ewald, nx, ny, nz] = self._old_system_forces['NonbondedForce'].getPMEParameters()
+            delta = self._old_system_forces['NonbondedForce'].getEwaldErrorTolerance()
+            r_cutoff = self._old_system_forces['NonbondedForce'].getCutoffDistance()
+            sterics_energy_expression, electrostatics_energy_expression = self._nonbonded_custom_ewald(alpha_ewald, delta, r_cutoff)
+        else:
+            raise Exception("Nonbonded method %s not supported yet." % str(self._nonbonded_method))
+        sterics_energy_expression += self._nonbonded_custom_sterics_common()
+        electrostatics_energy_expression += self._nonbonded_custom_electrostatics_common()
+
+        sterics_mixing_rules, electrostatics_mixing_rules = self._nonbonded_custom_mixing_rules()
+
+        # Create CustomNonbondedForce to handle interactions between alchemically-modified atoms and rest of system.
+        electrostatics_custom_nonbonded_force = openmm.CustomNonbondedForce("U_electrostatics;" + electrostatics_energy_expression + electrostatics_mixing_rules)
```

Comment:
Where do you call `electrostatics_custom_nonbonded_force.setNonbondedMethod()` and `sterics_custom_nonbonded_force.setNonbondedMethod()` to ensure these match their original reference `NonbondedForce` nonbonded methods?